test/test-clone.sh

```diff
@@ -144,12 +144,14 @@ begin_test "clone ClientCert"
   set -e
   reponame="test-cloneClientCert"
   setup_remote_repo "$reponame"
+  pwd
   out="$(clone_repo_clientcert "$reponame" "$reponame")"
```

Comment:
My hunch is that the directory change isn't happening since `clone_repo_clientcert` is being called from a sub-shell, `$()`.

If `clone_repo_clientcert` doesn't generate it's own log files, I'd just `| tee out.log`, and change the grep to:

```sh
if [ "0" -ne "$(grep -c 'client-cert-mac-openssl' < out.log)" ]; then
  # ...
fi
```